lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -94,7 +95,7 @@ public static RemoteTimelockServiceAdapter create(
 
     @Override
     public long getFreshTimestamp() {
-        return getFreshTimestamps(1).getLowerBound();
+        return conjureTimelockService.getFreshTimestamp().get();
```

Comment:
Checked that we already have logic in `RBTS` that calls `getFreshTimestamp` if size is 1, so this is actually an improvement